### PR TITLE
[21709] - Fix for "beam_PostCommit_Java_ValidatesRunner_Samza Failing"

### PR DESCRIPTION
**Pull request by fernando-wizeline**
Samza runner is also executing the BoundedReadFromUnboundedSourceTest test.
Adding it to Samza's exclusion list.

Bug reported here [BUG21709](https://github.com/apache/beam/issues/21709) and the CI error is here [Failed BoundedReadFromUnboundedSourceTest.testTimeBound](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/9581/testReport/org.apache.beam.sdk.io/BoundedReadFromUnboundedSourceTest/testTimeBound/)

@kennknowles or @damccorm, could you help me reviewing this?

Thanks!
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: runners/samza/build.gradle [hunk 1]
```diff
@@ -110,6 +110,8 @@ def sickbayTests = [
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testEncodingIOException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingNPException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingIOException',
+        // https://issues.apache.org/jira/browse/BEAM-6354
```
Comment (damccorm) [inline runners/samza/build.gradle:113]:
```suggestion
        // https://github.com/apache/beam/issues/19344
```

File: runners/samza/build.gradle [hunk 2]
```diff
@@ -110,6 +110,8 @@ def sickbayTests = [
...
+        'org.apache.beam.sdk.io.BoundedReadFromUnboundedSourceTest.testTimeBound',
```
Comment (damccorm) [inline runners/samza/build.gradle:114]: Is there an example of this failing in CI/how did we discover this issue? If there is, could you link it in the PR description?

File: runners/samza/build.gradle (hunk 1 above)
Comment (damccorm) [reply runners/samza/build.gradle:113]: I know the BEAM-XYZ is the convention here, but a change to update all references is currently in flight.

Comment (fernando-wizeline) [reply runners/samza/build.gradle:113]: Oh, sorry about that!

File: runners/samza/build.gradle (hunk 2 above)
Comment (fernando-wizeline) [reply runners/samza/build.gradle:114]: It was reported here [BUG21709](https://github.com/apache/beam/issues/21709) and the CI error is here [Failed BoundedReadFromUnboundedSourceTest.testTimeBound](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/9581/testReport/org.apache.beam.sdk.io/BoundedReadFromUnboundedSourceTest/testTimeBound/)

**Review by damccorm — APPROVED** [review]
Thanks - LGTM but you'll need a committer to merge once its out of draft

**Comment by fernando-wizeline** [comment]
> Thanks - LGTM but you'll need a committer to merge once its out of draft

Thanks a lot!

**Comment by kennknowles** [comment]
oh wait that is probably disabled

**Comment by kennknowles** [comment]
https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/9623/

I wonder if we can still make it have a PR status even from the postcommit build.

**Comment by kennknowles** [comment]
It seems the pull ref spec for this job is not adequate for getting the PRs. I will see if I can fix that.

**Comment by kennknowles** [comment]
Ran `./gradlew :runners:samza:validatesRunner --scan` locally and it passed: https://gradle.com/s/fu7ldgsrjjimo

**Review by kennknowles — APPROVED** [review]

**Comment by kennknowles** [comment]
I just also want to alert @xinyuiscool @kw2542 about this in case it is a test that should be passing.

**Pull request merged**